server/constants/order_status.js

```diff
@@ -5,16 +5,20 @@
  * pending -> active (subscription)
  * pending -> active -> cancelled (subscription)
  * pending -> cancelled
- * pending -> rejected
  * pending -> expired
  */
 
 export default {
-  PENDING: 'PENDING',
-  PAID: 'PAID',
-  ACTIVE: 'ACTIVE',
-  CANCELLED: 'CANCELLED',
-  REJECTED: 'REJECTED',
-  ERROR: 'ERROR',
-  EXPIRED: 'EXPIRED',
+  NEW: 'NEW', // New default state since August 2020
+  REQUIRE_ACTION: 'REQUIRE_ACTION', // For Strong Customer Authentication ("3D Secure")
```

Comment:
Not a fan of this one because it's not really explicit, a pending order always require an action. For example, a pending manual bank transfer also requires an action (from the client - to pay, and from the host admin - to confirm the payment).

If it's about `3D_SECURE`, I think something like one of the following may be more explicit:
- `REQUIRE_3D_SECURE`
- `REQUIRE_3D_SECURE_CONFIRMATION`
- `REQUIRE_CLIENT_CONFIRMATION`

Reply:
I picked this term because I followed Stripe's terminology https://stripe.com/docs/api/payment_intents/object#payment_intent_object-status I agree it doesn't look good out of this context.

- "Confirmation" means something else in the Stripe context.
- "3D_SECURE" seems a bit too specific, it's rarely used

In the end, `REQUIRE_CLIENT_CONFIRMATION` could work.

Reply:
+1 to `REQUIRE_CLIENT_CONFIRMATION`